src/rcl/molecules/SideBar.tsx

```diff
@@ -37,7 +37,7 @@ const SideBar = ({ navigation, className }: Props) => {
           <nav aria-label="Sidebar" className="flex flex-col">
             {navigation.map(({ name, href, isActive }) => (
               <RouteItem
-                key={href}
```

Comment:
The href is occasionally undefined in the case of having a non-linkable routeItem. So switch the key to the name attribute.